src/app/components/ReadTime/index.tsx

```diff
@@ -0,0 +1,22 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import styles from './index.styles';
+
+type ReadTimeProps = {
+  readTimeValue: number | undefined;
+  className?: string;
+};
+
+const ReadTime =({ readTimeValue, className }: ReadTimeProps) => {
+    if (readTimeValue === undefined) return false;
+
```

Comment:
Let's also only display this on local/test, which means we could merge this PR (and continue to iterate) and it won't be displayed to the audience.


```suggestion
if (isLive()) return null;
```